tasks/main.yml

```diff
@@ -23,21 +23,88 @@
     state: present
   notify: "Restart Fluentbit."
 
-- name: "Create needed directory."
+- name: "Create group td-agent-bit"
+  group:
+    name: td-agent-bit
+    state: present
+    system: True
+
+- name: "Create user with adm group"
+  user:
+    name: td-agent-bit
+    create_home: False
+    group: td-agent-bit
+    state: present
+    groups: adm
+    system: True
+    append: True
+
+- name: Ensure unit directory is present
+  file:
+    path: /etc/systemd/system/td-agent-bit.service.d
+    state: directory
+    mode: '0755'
+
+- name: Add unit override.conf file
+  copy:
+    src: override.conf
+    dest: /etc/systemd/system/td-agent-bit.service.d/override.conf
+
+- name: Add lua file
+  copy:
+    src: functions.lua
+    dest: /etc/td-agent-bit/functions.lua
+
+- name: "Create run directory."
   file:
     path: "{{ fluentbit_dbs_path }}"
     state: directory
     mode: 0755
+    owner: td-agent-bit
+    group: td-agent-bit
   when: fluentbit_dbs_path | length > 0
 
+- name: "Create logs directory."
+  file:
+    path: "{{ fluentbit_log_path }}"
+    state: directory
+    mode: 0755
+    owner: td-agent-bit
+    group: td-agent-bit
+  when: fluentbit_log_path | length > 0
+
 - name: "Enable Fluentbit service."
   service:
     name: "{{ fluentbit_svc_name }}"
     enabled: True
 
+- name: "Create Fluentbit parsers."
+  template:
+    src: custom_parsers.conf.j2
+    dest: /etc/td-agent-bit/custom_parsers.conf
+    mode: 0644
+
 - name: "Create main Fluentbit configuration."
   template:
     src: td-agent-bit.conf.j2
     dest: /etc/td-agent-bit/td-agent-bit.conf
     mode: 0644
   notify: "Restart Fluentbit."
+
+- name: Allow dd-agent to monitor fluentbit on port 2020
+  ansible.builtin.iptables:
```

Comment:
ansible.builtin.iptables: this module has pre-requisits and I'm not sure we have iptables everywhere (on prod/rax-priv envs we have only security groups no iptables). it deserves to test this situation

Reply:
Right, it isn't present on priv env, check added. 

Need to think how to do it in these envs

Reply:
As per swann's comment, is it ok?